lib/utilities.js

```diff
@@ -1,5 +1,42 @@
 'use strict';
 
+/**
+ * Constants.
+ */
+var CASE_SENSITIVE_TAGS = [
```

Comment:
Can we move this to its own file `lib/constants.js`?

Also can we update the JSDoc:
```js
/**
 * SVG elements, unlike HTML elements, are case-sensitive.
 *
 * @see {@link https://developer.mozilla.org/docs/Web/SVG/Element#SVG_elements_A_to_Z}
 */
```